### PR TITLE
feat: <Modal> components - data-testid attributes

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/3532714909/pulses/4493314347

### CODE REVIEW

File: src/tests/constants.ts [hunk 1]
```diff
@@ -92,7 +92,10 @@ export enum ComponentDefaultTestId {
   LIST = "list",
   LIST_ITEM = "list-item",
   LIST_TITLE = "list-title",
-  MODAL_BUTTONS = "modal-buttons",
+  MODAL = "monday-dialog-container",
```
Comment (shlomitc) [inline src/tests/constants.ts:95]:
```suggestion
  MODAL = "modal-dialog-container",
```

File: src/tests/constants.ts [hunk 2]
```diff
@@ -92,7 +92,10 @@ export enum ComponentDefaultTestId {
...
+  MODAL_OVERLAY = "monday-modal-overlay",
```
Comment (shlomitc) [inline src/tests/constants.ts:96]:
```suggestion
  MODAL_OVERLAY = "modal-overlay",
```

File: src/tests/constants.ts (hunk 1 above)
Comment (SergeyRoyt) [reply src/tests/constants.ts:95]: But it's a breaking change

File: src/tests/constants.ts [hunk 3]
```diff
@@ -92,7 +92,10 @@ export enum ComponentDefaultTestId {
...
+  MODAL_CONTENT = "modal-content",
+  MODAL_HEADER = "monday-header",
```
[the remark this answers is not in the document]
Comment (SergeyRoyt) [reply src/tests/constants.ts:98]: Fixed

File: src/tests/constants.ts (hunk 2 above)
Comment (SergeyRoyt) [reply src/tests/constants.ts:96]: But it's a breaking change

File: src/tests/constants.ts (hunk 1 above)
Comment (shlomitc) [reply src/tests/constants.ts:95]: bummer

**Review by shlomitc — APPROVED** [review]

**Pull request merged**